tests/e2e/jest.config.js

```diff
@@ -3,12 +3,15 @@
  * https://jestjs.io/docs/en/configuration.html
  */
 
+process.env.JEST_PLAYWRIGHT_CONFIG = 'jest-playwright.config.js';
+
 module.exports = {
-	preset: 'jest-puppeteer',
+	preset: 'jest-playwright-preset',
 	globalTeardown: './lib/global-teardown.js',
 	setupFilesAfterEnv: [
 		'jest-allure/dist/setup',
 		'<rootDir>/lib/setup-env.js',
-		'expect-puppeteer',
+		'expect-playwright',
 	],
+	testRunner: 'jasmine2',
```

Comment:
I understand that this line was added basically to make tests work. But I wonder if this is an optimal solution.

From what I gathered, with playwright we have a few options for the test runner:

- jasmine2 (which is used in the current configuration)
- jest-circus
- playwright-test

Could you talk about the pros and cons of these as well? I know that `circus` supports test re-run which I find really appealing. Also I wonder what benefits `playwright-test` could provide.

Reply:
Yes, this was a fix for an error thrown by jest-allure library. I don't really understand how come it wasn't failing before, with the Puppeteer suite of dependencies.
This was the only working solution I could find, based on this: https://github.com/zaqqaz/jest-allure/issues/8
It's not ideal, it was the way to move forward while keeping the Allure reports.
I will look into other options.

Reply:
Oh, I got it. I thought the source of this error was our own usage of `jasmine`. I guess we would need to move away from `jest-allure` if we'd like to switch to `circus` runner.

I wonder if there are alternatives to `jest-allure` that would work with `circus` runner.